VRDR/DeathRecord.csproj

```diff
@@ -13,9 +15,13 @@
   <ItemGroup>
     <PackageReference Include="Hl7.Fhir.R4" Version="1.6.0" />
     <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
-    <!-- <PackageReference Include="Vsxmd" Version="1.4.2">
+    <PackageReference Include="Vsxmd" Version="1.4.5">

```

Comment:
I don't think we want a dependency on Vsxmd to actually ship (we don't want users of our package to pull in this dependency), so I'd suggest 1) commenting it out by default, and 2) including a note on that in the section you added on generating the file